src/server/puppet-implementation.ts

```diff
@@ -435,9 +436,17 @@ export function puppetImplementation (
 
       try {
         const contactId = call.request.getContactId()
+        const callOptions = call.request.getOptions()
```

Comment:
I'd like to unfold all data to the root of the request:

```ts
const roomId = call.request.getRoomId()
const contactId = call.request.getContactId()
const hello = call.request.getHello()
```

If we move the data to root, then the `contactId` will be conflicted.

I'm thinking about using `XXXContactId` && `XXXRoomId` to replace the current name.

Reply:
As I posted here: https://github.com/wechaty/grpc/pull/122#discussion_r592859429, I think it would be better to wrap it into an message object, which will give us better consistency between puppet implementation and grpc proto.

Reply:
How about using `InviterContactId` && `InviterRoomId` to replace the current name?

BTW, if we wrap it into a message object, it should be mark as `optional`, but maybe we can not do that.